templates/prometheus.systemd.erb

```diff
@@ -12,6 +12,7 @@ ExecStart=<%= scope.lookupvar('prometheus::bin_dir') %>/prometheus \
 ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
 Restart=always
+LimitNOFILE=1000000
```

Comment:
Did you add this on purpose or was it planned for the other PR. I think this change isn't related, so please undo it.

Reply:
ah sorry, my mistake